db-engine/src/main/java/com/splicemachine/db/impl/sql/catalog/SYSCONSTRAINTSRowFactory.java

```diff
@@ -198,9 +195,6 @@ public ExecRow makeRow(boolean latestVersion, TupleDescriptor td,TupleDescriptor
 		/* 6th column is STATE (char(1)) */
         row.setColumn(SYSCONSTRAINTS_STATE,new SQLChar(isEnabled?"E":"D"));
 
-		/* 7th column is REFERENCED */
-        row.setColumn(SYSCONSTRAINTS_REFERENCECOUNT,new SQLInteger(referenceCount));
-
```

Comment:
Should we keep writing something here to prevent issues if we rollback to a previous version?
Say we upgrade to this version, create a constraint, we write only 6 values. There's an issue with the upgrade, so we rollback to the previous version. When deserializing the descriptor for the new constraint we try to read the 7th column but fail.

Reply:
I recommend not to drop the column even it is not used

Reply:
I've rolled back my changes. The field will not be dropped.